README.md

```diff
@@ -1 +1,88 @@
-# Metrics-webApp
+
+# CrytoCoin
+
+- CryptoCoin is an online web application that provides informations about the crypto currencies. Users can check real-time crypto prices and select them by categories. Build with React, Redux and Bootstrap.
+
+## ⚒️ Built With
+- React, Redux, Jest
+- Bootstrap
+- Git Workflow, Git, GitHub
+
+## Live Demo
+
+[Click here](https://deploycryptocoins.herokuapp.com/) 
+
+
+## Getting Started
+
+### To have a copy of this project running on your local machine, follow these commands:
+
+- Clone this repo to your local machine
+  -  `git clone git@github.com:tarikbouari/Metrics-App`
+  - Open the folder "Metrics-App `npm install` to install all necesary scripts,
+  - Run from your termminal, `npm start` to open the project in your browser.
+
+**This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- [NodeJS](https://nodejs.org/en/docs/)
+
+> As an asynchronous event-driven JavaScript runtime, Node.js is designed to build scalable network applications. In the following "hello world" example, many connections can be handled concurrently. Upon each connection, the callback is fired, but if there is no work to be done, Node.js will sleep.
+
+- [Text Editor(VScode Recommended)](https://code.visualstudio.com/)
+
+> Visual Studio Code is a free coding editor that helps you start coding quickly. Use it to code in any programming language, without switching editors.
+
+### Setup
+
+**Run from your terminal:**
+
+- `npx create-react-app Metrics-App`
+
+**cd into Metrics-App folder, then run from your terminal:**
+
+- `npm start`
+
+### Usage
+
+**When the above commands are successfully run, your browser should open on `localhost:3000`, if not, open your browser and load `127.0.0.1:3000` to see your default template loaded
+
+### Run tests
+
+**Run from your terminal:**
+
+- `npm test`
+
+_This displays the results of the test suites already written on the project to test for various functionalities and UI implementations. Feel free to edit the UI components and re-run the tests to verify their authenticity_
+
+### Deployment
+
+> coming soon :smile:
+
+## Authors
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.